pkgs/development/node-packages/composition-v13.nix

```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-13_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-8_x"}:
```

Comment:
Not sure why this got changed.

Reply:
I was about to ask :)

Reply:
`node2nix` is called with the `--node-js-13` argument, which according to https://github.com/svanderburg/node2nix/blob/master/bin/node2nix.js#L134 is not implemented yet...

@marsam did you created this file manually? Is anything special preventing us to upstream that to `node2nix`?

Reply:
Created a proper issue to track that: https://github.com/NixOS/nixpkgs/issues/78044

Reply:
Created a proper issue to track that: https://github.com/NixOS/nixpkgs/issues/78044

Reply:
>@marsam did you created this file manually?

yes, I remember changing this line with `13_x`. Sorry about that!

Reply:
> Sorry about that!

No problem :) Let's fix this :+1: